src/style.css

```diff
@@ -1,3 +1,59 @@
+:root {
+  --main-blue-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 40px;
+}
+
+.navigation {
+  margin: 0;
+}
+
+.nav-link-list {
+  display: flex;
+  margin: 0;
+}
+
+.link-item {
+  list-style: none;
+  line-height: 60px;
+  position: relative;
+}
+
+.link-item:not(:last-child) {
+  margin-right: 20px;
+  margin-left: 20px;
+}
+
+.link-item a {
+  text-decoration: none;
+  color: #000;
```

Comment:
The color for the links is set to `#000`, which is black. Ensure that this provides enough contrast against the background for accessibility purposes.